README.md

```diff
@@ -77,6 +78,7 @@ Windows (x64)         | [Installer (exe)][win-x64-exe]<br>[Archive (zip)][win-x6
 Windows (x86)         | [Installer (exe)][win-x86-exe]<br>[Archive (zip)][win-x86-zip]   | [Installer (exe)][win-x86-exe-rel-21]<br>[Archive (zip)][win-x86-zip-rel-21]
 macOS (x64)           | [Archive (tar.gz)][osx-x64-tar]                                  | [Archive (tar.gz)][osx-x64-tar-rel-21]
 Linux (x64)           | [Archive (tar.gz)][linux-x64-tar]                                | [Archive (tar.gz)][linux-x64-tar-rel-21]
+Linux (arm32)         | [Archive (tar.gz)][linux-arm-tar]                                | (not available yet)
```

Comment:
Note to self: once we have a release/2.1 one build, we can update the README to include a link to this as well the release/2.1 channel column